controllers/reconcilers/kserve_metrics_dashboard_reconciler.go

```diff
@@ -184,13 +143,9 @@ func (r *KserveMetricsDashboardReconciler) createDesiredResource(ctx context.Con
 		},
```

Comment:
(optional) looks like this variable used several places 
```
isvc.Name + "-metrics-dashboard",
```
So I think it would be a good idea to use a variable like the following to make the code more readable and easier to maintain.to maintain.
```
metricsCMSuffix := "-metrics-dashboard"
metricsCMName := isvc.Name + metricsCMSuffix

configMap := &corev1.ConfigMap{
		ObjectMeta: metav1.ObjectMeta{
			Name:      metricsCMName,
			Namespace: isvc.Namespace,
		},
```